### PR TITLE
Loyalty Program page

### PR DESCRIPTION
**Pull request by l2succes**
# Things to finish

- [ ] Setup client-side part of isomorphic-relay 
- [ ] After logging in from force, it should redirect to /loyalty/inquiries
- [ ] Finish adding metadata tags to <head> of pages
- [ ] Add graphQL mutation to submit inquiries update
- [ ] Implement artwork selection UI
- [ ] Create "Thank you" page

### Development tools
- [ ] Need to fix source maps
- [ ] Hot reloading (Goal is to get to that level: [ultimate-hot-reloading](https://github.com/glenjamin/ultimate-hot-reloading-example))

### Steps to setup force/reaction-force dev environment

```
cd reaction-force
yarn link

cd ../force
yarn link reaction-force
```

_this creates a symlink of reaction-force in `force/node_modules/`_

To publish update to npm:
`yarn publish`

### CODE REVIEW

**Comment by alloy** [comment]
I can do the isomorphic-relay and dev tools tasks, so others please let me know if you’d like to do some of these instead or forever hold your ☮️

File: src/apps/loyalty/containers/inquiries/index.tsx [hunk 1]
```diff
@@ -1,27 +1,92 @@
...
+    if (!this.props.user) {
+      return []
+    }
+
+    const edges = this.props.user.artwork_inquiries_connection.edges || []
+    return edges.map(edge => {
+      const artwork = edge.node.artwork
+      return (
+        <InquiryContainer>
+          <Artwork key={artwork.id} artwork={artwork} />
```
Comment (l2succes) [inline src/apps/loyalty/containers/inquiries/index.tsx]: Btw,`artwork.id` needs to be a global id across the app. Should use `__id` from relay or another global id.

Comment (alloy) [inline src/apps/loyalty/containers/inquiries/index.tsx]: Why did you not switch to `__id`?

**Comment by alloy** [comment]
@l2succes Alright, this should be done enough for you to continue on. I think it’s probably best if the current state gets merged asap (so the branches don’t deviate more) and work on the other TODOs incrementally.

**Comment by l2succes** [comment]
@alloy ok awesome. I'll fix the merge conflict real quick

**Pull request merged**